CMakeLists.txt

```diff
@@ -86,7 +86,12 @@ if(NOT CMAKE_BUILD_TYPE AND NOT CMAKE_CONFIGURATION_TYPES)
     "MinSizeRel" "RelWithDebInfo")
 endif()
 
-include_directories(include src src/rml/include ${CMAKE_CURRENT_BINARY_DIR})
+
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}/src
```

Comment:
Is this actually needed? I think CMake compilation commands are always relative to ${CMAKE_CURRENT_SOURCE_DIR}